.github/workflows/dependencies.yml

```diff
@@ -29,12 +29,34 @@ jobs:
   check-bans:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
       - uses: dtolnay/rust-toolchain@stable
       - name: Install cargo-deny
         run: cargo install cargo-deny
-      - name: Check for banned and duplicated dependencies
-        run: cargo deny check bans
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.base_ref }}
+      - name: Cargo update (base)
+        run: cargo update
+      - name: Check for banned and duplicated dependencies (base)
+        run: |
+          set -o history
+          OUT_FILE=/tmp/deny-bans-base
```

Comment:
could you put the two `OUT_FILE` vars in an `env:` at the start of the job with different names?